src/Commands/NotificationQueuer.php

```diff
@@ -69,54 +69,24 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Que
    *
    * @param string $media_type_id
    *   The media type ID to import for.
+   * @param array $options
+   *   An array of command options.
+   *
+   * @option once Only process a single notification.
```

Comment:
I think this could be clearer. Isn't this meant to process a single_batch_ of notifications? On this line https://github.com/Lullabot/media_mpx/pull/106/files#diff-4459423a40e87d2a49044bd7b4e236a7R276 it looks like we have multiple notifications that get processed even when this param is set, it's just when it's set, it finished with that set and returns.

Reply:
Yep, no more need to while true hack it!

Good thought about "once" - I really mean "notification response".